christfellowshipchurch/web-app-v2
# Saving search input in discover

### About
This PR makes it so that when a user searches in discover and goes back after clicking on a result the search filter, input, and therefore results are still there.

### Test Instructions
1. Go to `/discover` on the testing link ([here](https://web-app-v2-git-saving-on-search-christ-fellowship-church.vercel.app/discover?c=on-demand-messages&s=)).
2. Try searching for something and changing the filters, open one of the resources on the results and then go back. The search results should be there.

### Screenshots
<img width="1184" alt="Screenshot 2023-11-22 at 2 06 41 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/4fedd57d-80c1-4c07-81c1-0b3ebb86a0b0">


### Closes Tickets
[CFDP-2791](https://christfellowshipchurch.atlassian.net/browse/CFDP-2791)

[CFDP-2791]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2791?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
